techLogger.js

```diff
@@ -72,47 +74,29 @@ function _makeLogger(prefix) {
             return updated;
         },
 
-        debug: function() {
-            _log("debug", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        debug: (...args) => _log("debug", _prefixLog(args, logPrefix)),
 
-        info: function() {
-            _log("info", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        info: (...args) => _log("info", _prefixLog(args, logPrefix)),
 
-        notice: function() {
-            _log("notice", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        notice: (...args) => _log("notice", _prefixLog(args, logPrefix)),
 
-        warn: function() {
-            _log("warning", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        warn: (...args) => _log("warning", _prefixLog(args, logPrefix)),
 
-        error: function() {
-            _log("error", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        error: (...args) => _log("error", _prefixLog(args, logPrefix)),
 
-        crit: function() {
-            _log("crit", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        crit: (...args) => _log("crit", _prefixLog(args, logPrefix)),
 
-        alert: function() {
-            _log("alert", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        alert: (...args) => _log("alert", _prefixLog(args, logPrefix)),
 
-        emerg: function() {
-            _log("emerg", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
+        emerg(...args) {
+            return _log("emerg", _prefixLog(args, logPrefix));
         },
 
-        createExpressLoggerStream: function(level) {
-            return {
-                write: function(message /*, encoding */ ) {
-                    _log(level, message);
-                }
-            };
+        createExpressLoggerStream(level) {
+            return { write: (message /*, encoding */ ) => _log(level, message) };
         },
 
-        splash: function(app, configuration) {
+        splash(app, configuration) {
             splash(this, app, configuration);
         }
```

Comment:
I don't really know... :switzerland: 

:one: I don't mind mixing the syntaxes.
But if we do so, we will have to pay attention and check if method contains `this` or not (this can lead to mistakes...)

:two: I we don't mix syntaxes, we can still use shorthand syntax...




In both case, we need to pay attention when we are writing object with methods
:one: Use arrow function when method is not using `this` and shorthand syntax otherwise
:two: Use shorthand syntax for all methods of objects containing at least one method with `this`...


Conclusion : I don't know... I'm okay to follow what will be chosen...